mslib/mscolab/server.py

```diff
@@ -616,6 +617,15 @@ def set_version_name():
     return jsonify({"success": True, "message": "Successfully set version name"})
 
 
+@APP.route("/edit_user_info", methods=["POST"])
```

Comment:
where is this used?

Reply:
When it is planned for 

![image](https://github.com/user-attachments/assets/54b61252-37f2-46ba-be39-ac597617e56c)

The function should replace the Change Avatar, by an Edit. 

e-mail won't be allowed to become changed

and for changing the username you need to verify that the new one was not taken,